.github/workflows/pypi.yml

```diff
@@ -65,7 +65,7 @@ jobs:
           python-version: ${{ matrix.pyver }}
       - name: Install python packages
         run: |
-          pip install pybind11 wheel
+          pip install pybind11<=2.13.1 wheel
```

Comment:
Could change to `pip install -r `requirements/build.txt` and lock pybind11 version in `build.txt`

Reply:
Good idea!